RceG3/ruckus.tcl

```diff
@@ -3,13 +3,17 @@ source -quiet $::env(RUCKUS_DIR)/vivado_proc.tcl
 
 # Load local Source Code
 loadSource -dir  "$::DIR_PATH/hdl/"
+
 loadSource -path "$::DIR_PATH/simlink/rtl/RceG3CpuSim.vhd"
-loadIpCore -dir  "$::DIR_PATH/coregen/processing_system7_0/"
+
+loadIpCore -dir       "$::DIR_PATH/coregen/processing_system7_0/"
+loadBlockDesign -path "$::DIR_PATH/coregen/pcie_root/pcie_root.bd"
+
 loadSource -sim_only -path "$::DIR_PATH/tb/rceg3_tb.vhd"
 
 # Check for submodule tagging
 if { [SubmoduleCheck {ruckus} {1.6.3} ] < 0 } {exit -1}
 if { [SubmoduleCheck {surf}   {1.7.4} ] < 0 } {exit -1}
 
 # Check for version 2016.4 of Vivado (or later)
-if { [VersionCheck 2016.4] < 0 } {exit -1}
\ No newline at end of file
+if { [VersionCheck 2016.4] < 0 } {exit -1}
```

Comment:
I don't think a min. 2016.4 is support any more.  Example: spirit:referenceId="RUNTIME_PARAM.SWVERSION">2017.4</spirit:configurableElementValue> in the zynq_10g_xaui.xci.

We could probably move this to 2017.4.  Have you've tried building with 2017.4 (or using newer version like 2018.2)?

Reply:
That was left over from before. I can update. I have built with 2017.4 and 2018.1.